requirements.txt.full

```diff
@@ -2,6 +2,7 @@
 sssom
 oaklib
 linkml
+babelon
```

Comment:
No objection to adding that.

Just, have you already tried building the ODK with it? We just need to be sure this addition does not create yet another mess with Python dependencies (e.g. if Babelon depends on on _foo_ > 1.2.3 and we have another Python packages that requires _foo <= 1.1.1).

Reply:
Yes I did and I had to make a new release! Our QC actually takes this into account.